examples/run/alpaka/CMakeLists.txt

```
@@ -4,21 +4,42 @@
 #
 # Mozilla Public License Version 2.0
 
-set(LIBRARIES vecmem::core traccc::io traccc::performance
-    traccc::core traccc::device_common traccc::alpaka alpaka::alpaka
-    traccc::options)
+set(EXTRA_LIBS "")
```

I personally prefer the formalism:

```suggestion
set(EXTRA_LIBS)
```

Though technically this one is fine as well.